src/client.js

```diff
@@ -23,8 +23,8 @@ class Client extends EventEmitter {
   sendMessage (id, service, text, file) {
     return iMessageSend(id, service != "iMessage" ? "sms" : "iMessage", text, file);
   }
-  sendGroupMessage (handles, text, file) {
-    return iMessageSendGroup(handles, text, file);
+  sendGroupMessage (id, text, file) {
+    return iMessageSendGroup(id, text, file);
```

Comment:
I think this requires also a change in https://github.com/matrix-hacks/matrix-puppet-imessage/blob/master/index.js#L196

This probably resolved the issue of  @ericmigi